src/javaRosa.js

```diff
@@ -823,6 +823,8 @@ define([
             widgetClass = options.baseWidgetClass || widgets.richTextarea,
             widget, $input;
 
+        options.language = language;
```

Comment:
Seems like a strange place to be setting an option, given that both `language` and `options` are passed into this function. Is there a precedent for this?